### PR TITLE
ignore update_all_packages when offline mode

### LINKED ISSUE #74: ovirt_engine_setup_offline and ovirt_engine_setup_update_all_packages discrepancy
I suggest to squash these into just a single option. You either want to update everything ro run in offline environment.
But having e.g. ovirt_engine_setup_update_all_packages=true and ovirt_engine_setup_offline=true doesn't make sense - you update all the packages and then setup fails because it's not allowed to install matching packages (in versionlock). Similarly the other way around.

It makes sense to only have a single option and derive a correct --offline parameter from it

### PR DESCRIPTION
**Pull request by mnecas**
Fixes #74
add ignore for `ovirt_engine_setup_update_all_packages` when `ovirt_engine_setup_offline` is True

WDYT @michalskrivanek

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ as ``ovirt_engine_setup_answer_file_path``.
 | ovirt_engine_setup_use_remote_answer_file | False             | If `True`, use answerfile's path on the remote machine. This option should be used if the installation occure on the remote machine and the answerfile is located on there also. |
 | ovirt_engine_setup_update_setup_packages | False              | If `True`, setup packages will be updated before `engine-setup` will be executed. Makes sense if Engine is already installed. |
 | ovirt_engine_setup_perform_upgrade    | False                 | If `True` this role is used to perform upgrade. |
-| ovirt_engine_setup_update_all_packages | True                 | If `True`, all packages will be updated before `engine-setup` will be executed. |
+| ovirt_engine_setup_update_all_packages | True                 | If `True`, all packages will be updated before `engine-setup` will be executed. If `ovirt_engine_setup_offline` is `True` the packages will not be updated. |
```
Comment (michalskrivanek) [inline README.md]: I would rephrase to use a positive case. I.e. explain that `ovirt_engine_setup_offline needs to be False otherwise it's not truly updated

**Comment by mnecas** [comment]
Yeah, I'm not the best person to write docs.
